### PR TITLE
toHTTP function moved from github.com/influxdata/ifql

### PR DESCRIPTION
**Pull request by docmerlin**
toHTTP function moved from github.com/influxdata/ifql
https://github.com/influxdata/ifql/pull/362

### CODE REVIEW

**Comment by pauldix** [comment]
I'd prefer this be renamed to simply `to`. It's hitting the InfluxDB API so it's not just a general HTTP endpoint.

From a user standpoint, I'd prefer that `to` has `addr` as an optional. If it's not provided then the to should be this InfluxDB server or cluster.

Also, would prefer that param name be `host` since `addr` is a Go idiom that I haven't seen elsewhere.

**Comment by docmerlin** [comment]
We were thinking multiple `toFoo` functions, and a `to` function that wrapped them and picked a sane one for the situation when a person just wanted to send data back to their own influx server, instead of doing something else with it.

File: query/functions/to_http.go [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+	isTag := make([]bool, len(colMetadatas))
+	isValue := make([]bool, len(colMetadatas))
+
+	for i, col := range colMetadatas {
+		isValue[i] = sort.SearchStrings(t.spec.Spec.ValueColumns, col.Label) < len(t.spec.Spec.ValueColumns) && t.spec.Spec.ValueColumns[sort.SearchStrings(t.spec.Spec.ValueColumns, col.Label)] == col.Label
+		isTag[i] = sort.SearchStrings(t.spec.Spec.TagColumns, col.Label) < len(t.spec.Spec.TagColumns) && t.spec.Spec.TagColumns[sort.SearchStrings(t.spec.Spec.TagColumns, col.Label)] == col.Label
+	}
+
+	var err error
+	go func() {
```
Comment (nathanielc) [inline query/functions/to_http.go]: To your comment above.

A block is only valid for the duration of the Process call. It is fine to read it asynchronously with making the HTTP request. But you need to guarantee that it is not read once the Process function returns.

Comment (docmerlin) [reply query/functions/to_http.go]: Thanks.  Added a wg.

**Pull request merged**